data/colors/vars/component_light.ts

```diff
@@ -147,5 +147,20 @@ export default {
   underlinenav: {
     icon: get('fg.subtle'),
     borderHover: get('neutral.muted')
+  },
+
+  actionListItem: {
+    inlineDivider: alpha(get('border.default'), 0.48),
+    
+    default: {
+      hoverBg: alpha(get('scale.gray.2'), 0.32),
+      activeBg: alpha(get('scale.gray.2'), 0.48),
+      selectedBg: alpha(get('scale.gray.2'), 0.24)
+    },
+    danger: {
+      hoverBg: alpha(get('danger.subtle'), 0.64),
```

Comment:
Is there a reason the `light` theme uses `danger.*` variables and the `dark` theme uses `scale.red.*` variables?

Reply:
@colebemis it looks like `danger.subtle` on `light` theme uses a fully opaque light red, while `danger.subtle` on dark mode uses a 15% opacity red. Would `alpha()` on a color with transparency like that  change its current opacity to a different value?